src/patch_parser.py

```diff
@@ -458,9 +483,11 @@ def _parse_patch_file_added_chunk(
 def _parse_patch_file_removed_chunk(
         lines: InputSource,
         gerrit_orig_line: int,
-        gerrit_new_line: int) -> Tuple[int, int, PatchFileChunkLineMap]:
+        gerrit_new_line: int,
+        parser_state: HunkParserState) -> Tuple[int, int, PatchFileChunkLineMap]:
     in_start = lines.line_number()
     while lines[0] and lines[0][0] == '-':
+        parser_state.add_deleted_line()
```

Comment:
nit: I'd slightly prefer the `lines.consume()` comes first and this integer increment goes down with the rest (`gerrit_orig_line += 1`)